tests/robustness/multiclient_test/framework/snapshotter.go

```diff
@@ -67,6 +72,23 @@ func (mcs *MultiClientSnapshotter) ConnectOrCreateRepo(repoPath string) error {
 	return err
 }
 
+// setCacheSizeLimits sets the content cache size limits for an existing repository
+// the repository server is connected to.
+func (mcs *MultiClientSnapshotter) setContentCacheSizeLimit(contentCacheSize int) error {
```

Comment:
nit: perhaps we should rename these functions to include a hint on what the integer value argument represents: `setContentCacheSizeLimitMB` and/or rename the argument name to include the MB unit

Reply:
done